src/status_im/ui/screens/browser/views.cljs

```diff
@@ -112,7 +112,7 @@
 (def resources-to-permissions-map {"android.webkit.resource.VIDEO_CAPTURE" :camera
                                    "android.webkit.resource.AUDIO_CAPTURE" :record-audio})
 
-(views/defview request-resources-panel [resources url]
+(views/defview request-resources-panel [resources url wb]
```

Comment:
could we keep `webview-ref` ?  `wb` is kinda hard to understand what it is from the name

Reply:
sure